src/js/sdk.tsx

```diff
@@ -130,10 +133,10 @@ export function init(passedOptions: ReactNativeOptions): void {
   });
   initAndBind(ReactNativeClient, options);
 
-  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-explicit-any
-  if (getGlobalObject<any>().HermesInternal) {
+  if (isHermesEnabled()) {
```

Comment:
Can we move this to `ReactNativeInfo` as well?
Wondering if there are more places in the code that would make sense to be in the `ReactNativeInfo`

Reply:
Yes, I'll move this.

Do you have anything in mind? I have checked all the tags that the SDK is adding now automatically and I don't there is anything there that should be included.

Very likely in the future the RN version, maybe Hermes version will be here but that is another task.

Reply:
Okay, than its all.